homeassistant/components/mikrotik/device_tracker.py

```diff
@@ -149,19 +150,16 @@ def update_device_tracker(self):
                 if attr in device and device[attr] is not None:
                     attrs[slugify(attr)] = device[attr]
 
-            attrs["scanner_type"] = self.method
-            attrs["scanner_host"] = self.host
-            attrs["scanner_hostname"] = self.hostname
```

Comment:
Is this removal intended?

Is it a breaking change or have we not released these yet?

Reply:
Didn't mean to remove that. This has not been released yet.